status-im/status-react
# feat: style delivery states

fixes #14800 

### Summary

This issue implements style to delivery states as per Figma designs. Delivery states are shown when user single-taps a message, and it is shown for just 3 seconds, then it disappears.

<img width="300" src="https://user-images.githubusercontent.com/18485527/215891286-6958ece9-c3c9-4709-84f9-400fdbfed543.png">
<img width="300" src="https://user-images.githubusercontent.com/18485527/215891280-49ac7022-96cf-48a9-a416-39b5cb9cec08.png">
<img width="300" src="https://user-images.githubusercontent.com/18485527/215891272-ccea7a8e-0ca8-4380-95e0-d3c1210b88b2.png">

### Testing notes

- Sometimes it may be difficult to find and test all message outgoing states, I used waku1 fleet for testing but should work with waku2 as well
- The `sending` state, which makes the message to have opacity, may happen for a very very short time
- This PR does not include handling error states because it is out of scope of this issue
- As messages UI are built by different building blocks making the whole thing and there is pending work on other messages components, testing should be focused on just delivery states UI, take in mind that the whole messages design and some spacing may differ of how they should look when all elements are implemented as they should

#### Platforms

- Android
- iOS

#### Areas that maybe impacted

##### Functional

- 1-1 chats
- public chats
- group chats

### Steps to test

- Open Status
- Create a chat or community with other users
- Send messages
- Verify delivery states UI 

status: ready
